llvm/test/CodeGen/NVPTX/tensormap-cp-fence-proxy.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_90 -mattr=+ptx83 | FileCheck --check-prefixes=CHECK %s
```

Comment:
I'd suggest using automated check generation. While manual checks here will check for the correct instructions, they do not verify pointer argument order.

I'd also add a ptxas run command and verify that ptxas does accept the PTX we generate for these intrinsics.